vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+
+```{r}
+fit_sex_district <- epidist(
+  data = obs_prep,
+  formula = brms::bf(
+    mu ~ 1 + sex + (1 | district),
+    sigma ~ 1 + sex + (1 | district)
+  ),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+Along with looking at the `summary`, we may also use the `brms::ranef` function to look at the estimates of the random effects:
+
+```{r}
+summary(fit_sex_district)
+brms::ranef(fit_sex_district)
+```
+
+## Posterior expectations
+
+To go beyond simple summaries of the fitted model, we recommend using the `tidybayes` package.
+For example, to obtain the posterior expectation of the delay distribution, under no censoring or truncation, we may use the `modelr::data_grid` function in combination with the `tidybayes::add_epred_draws` function.
+The `tidybayes::add_epred_draws` function uses the `posterior_epred_latent_lognormal` function implemented in `epidist` for the `latent_lognormal` custom `brms` family.
+First, for the model with no stratification (Figure \@ref(fig:epred-base)):
+
+```{r}
+epred_draws <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(NA) |>
+  dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA) |>
+  add_epred_draws(fit, dpar = TRUE)
+```
+
+(ref:epred-base) The fitted posterior expectation of the delay distribution for a model with no stratification.
+
+```{r epred-base, fig.cap="(ref:epred-base)"}
+epred_draws |>
+  ggplot(aes(x = .epred)) +
+  tidybayes::stat_slab() +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+Next, for the sex stratified model, we illustrate the higher mean of men as compared with women (Figure \@ref(fig:epred-sex)):
+
+```{r}
+# Probably want functionality for users not to have to do this themselves
+(newdata_sex <- obs_prep |>
+   as.data.frame() |>
+   modelr::data_grid(sex) |>
+   dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA))
+
+epred_draws_sex <- tidybayes::add_epred_draws(newdata_sex, fit_sex, dpar = TRUE)
+```
+
+(ref:epred-sex) The fitted posterior expectation of the delay distribution for a model with sex stratification.
+
+```{r epred-sex, fig.cap="(ref:epred-sex)"}
+epred_draws_sex |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .epred, y = sex)) +
+  tidybayes::stat_halfeye() +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+Finally, the sex-district stratified model is shown in Figure \@ref(fig:epred-sex).
+Note that these results are questionable, and suggest that further would should be done refining the prior distributions for the model.
+Part of the issue is that some sex-district strata have quite limited data.
+
+```{r}
+epred_draws_sex_district <- obs_prep |>
```

Comment:
I suggest you cut the x axis you make this easier to visualise (and say you have done this)

Reply:
[4035227](https://github.com/epinowcast/epidist/pull/52/commits/40352278c509ec647c7255e233a6ffc6b6a57472)